source/gui/settingsDialogs.py

```diff
@@ -3320,33 +3320,58 @@ class RemoteSettingsPanel(SettingsPanel):
 	title = _("Remote")
 	helpId = "RemoteSettings"
 
-	def makeSettings(self, sizer):
+	def makeSettings(self, sizer: wx.BoxSizer):
 		self.config = config.conf["remote"]
 		sHelper = guiHelper.BoxSizerHelper(self, sizer=sizer)
 
-		self.playSounds: wx.CheckBox = sHelper.addItem(
-			# Translators: A checkbox in Remote settings to set whether sounds play instead of beeps.
-			wx.CheckBox(self, label=_("&Play sounds instead of beeps")),
+		self.enableRemote = sHelper.addItem(
+			# Translators: Label of a checkbox in Remote's settings
+			wx.CheckBox(self, label=pgettext("remote", "Enable Remote Access")),
+		)
+		self.enableRemote.Bind(wx.EVT_CHECKBOX, self._onEnableRemote)
+		self.bindHelpEvent("RemoteEnable", self.enableRemote)
+
+		remoteSettingsGroupSizer = wx.StaticBoxSizer(
+			wx.VERTICAL,
+			self,
+		)
+		self.remoteSettingsGroupBox = remoteSettingsGroupSizer.GetStaticBox()
+		remoteSettingsGroupHelper = guiHelper.BoxSizerHelper(self, sizer=remoteSettingsGroupSizer)
+		sHelper.addItem(remoteSettingsGroupHelper)
+
+		self.playSounds: wx.CheckBox = remoteSettingsGroupHelper.addItem(
```

Comment:
Usages of `BoxSizerHelper.addItem` should return the correct type.

I think all of these type hints might be redundant - do they flag a pyright error?